docs/migration/3_0.md

```diff
@@ -65,6 +65,11 @@ There's no automation (schematics) planned for this change.
 
 - ConverterService constructor is now using `UnifiedInjector` instead of standard `Injector` 
 
+### CartItemComponent lost members
+`CartItemComponent` lost members: 
+- `@Output()` `view` - instead use `[cxModal]` directive to close modal on click link
```

Comment:
Could we switch `click` and `link`? It sounds strange to me.

Reply:
Done